### PR TITLE
Update branding on /switch (#7608)

### LINKED ISSUE #7608: Update branding on /switch page
switch

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- new switch page CSS and assets
- convert subnav to protocol
- add email icon to dark icon sprite

## Issue / Bugzilla link

Fix #7608 

## Testing

http://localhost:8000/en-US/firefox/switch/
- RTL
- in Chrome

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Note: The footer section talks about still being able to use Chrome and sharing that you've switched to Firefox and it's weird to smoosh those two things together. This PR, however, does not have any string changes so the awkwardness remains.

File: bedrock/firefox/templates/firefox/switch.html [hunk 1]
```diff
@@ -16,61 +18,70 @@
...
+    </div>
+  </section>
 
-        {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+  <section class="c-share">
+    <div class="mzp-l-content">
 
-        <footer>
-          <p>{{ _('You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.') }} {{ _('Share with your friends how to switch to Firefox') }}</p>
+      {% if l10n_has_tag('switch_201908') %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/switch.html]: Note to make sure we expose these new strings for translation, post launch.

File: bedrock/firefox/templates/firefox/switch.html [hunk 2]
```diff
@@ -16,61 +18,70 @@
...
-              {# L10n: the string below is the introduction of the email referral message body. #}
-              {% set email_intro = _('Hey,')|mailtoencode %}
-              {# L10n: the string below is the first sentance of the email referral message body. #}
-              {% set email_start = _('Firefox makes switching from Chrome really fast. I like it a lot, and you should try it.')|mailtoencode %}
-              {# L10n: the string below is the second sentance of the email referral message body. At the end of the string after the colon, a URL will be added automatically. #}
-              {% set email_end = _('Check it out and let me know what you think:')|mailtoencode %}
+      {% set share_url = settings.CANONICAL_URL + canonical_path %}
+      <ul class="c-share-options">
+        <li>
+          <a class="facebook" data-link-type="social share" data-link-name="Facebook" href="https://www.facebook.com/sharer/sharer.php?s=100&p[url]={{ share_url|urlencode }}">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/switch.html]: My Facebook Container extension blocks this link:

image

Considering we no longer promote on Facebook any longer or advertise there, should we consider removing this link?

File: bedrock/firefox/templates/firefox/switch.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/switch.html]: Merged. Here is a note to remind you about your note.

Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/switch.html]: Ooops, still catching up on emails. There's a string change to the string change. I'll submit a new PR.

**Pull request merged**